src/Analyzers/Core/Analyzers/IBuiltInAnalyzer.cs

```diff
@@ -2,10 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.CodeAnalysis.Options;
-
 namespace Microsoft.CodeAnalysis.Diagnostics
 {
+#if CODE_STYLE
```

Comment:
Any code accessing `OptionSet` in our shared projects is going to need such #if until we can move all our analyzer/fixers linked into Features.dll to also move to `AnalyzerConfigOptions` (tracked by https://github.com/dotnet/roslyn/issues/41462). Even then we might have cases where we need these when a helper method in Workspace layer needs to operate invoked outside the analyzer/fixer context and needs to operate on `OptionSet`, which same helper is required by some analyzer/fixer and must operate on `AnalyzerConfigOptions`